express/code/blocks/quotes/quotes.css

```diff
@@ -70,24 +191,47 @@
   font-weight: 900;
 }
 
+.quotes.carousel .quote .content::before {
+  display: none;
+}
+
 .quotes .quote p {
   margin: 32px 0;
 }
 
+/* Author section styles */
 .quotes .quote .author {
   display: flex;
-  flex-wrap: nowrap;
-  margin-top: 16px;
+  flex-direction: column;
   align-items: center;
-  align-self: baseline;
+  justify-content: center;
+  width: 100%;
+  margin-top: 0;
 }
 
+.quotes.carousel .quote .author {
+  display: flex;
+  margin-top: 0;
+  align-items: center;
+}
+.quotes.carousel .quote .author > p {
+  display: none;
+}
 .quotes .quote .author hr {
   display: none;
 }
 
-.quotes .quote .author .image {
-  padding-right: 10px;
+.quotes .quote .author .summary p:first-of-type {
+  font-size: var(--body-font-size-s);
+}
+
+.quotes .quote .author .summary p:nth-of-type(2) {
+  font-size: 16px;
+  font-weight: bold;
+}
+
+.quote .content {
```

Comment:
Looks like we'll need more specificity here as this causes small regressions 

https://uk-campaign-updated-quotes-carousel--express-milo--adobecom.hlx.page/express/discover/quotes/success

vs

https://main--express-milo--adobecom.hlx.page/express/discover/quotes/success